app/services/account_reset/create_request.rb

```diff
@@ -38,7 +44,11 @@ def notify_user_by_sms_if_applicable
       cancel_link = Rails.application.routes.url_helpers.account_reset_cancel_url(
         token: user.account_reset_request.request_token,
       )
-      Telephony.send_account_reset_notice(to: phone, cancel_link: cancel_link)
+      @telephony_response = Telephony.send_account_reset_notice(to: phone, cancel_link: cancel_link)
+    end
+
+    def extra_analytics_attributes
+      @telephony_response&.extra&.slice(:request_id, :message_id) || {}
```

Comment:
I was considering re-mapping `message_id` to `sms_message_id` in a few places, but wasn't sure if that would be more or less confusing